generic/utils/ras.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+import os
+import commands
+from avocado import Test
+from avocado import main
+from avocado.utils import process
+from avocado.utils.software_manager import SoftwareManager
+
+
+class RASTools(Test):
+
+    def setUp(self):
+        Arch = commands.getoutput("arch")
+        if "ppc" not in Arch:
+            self.skip("supported only on Power platform")
+        sm = SoftwareManager()
+        if not sm.check_installed("ppc64-diag") and not sm.install("ppc64-diag"):
+            self.error('ppc64-diag is needed for the test to be run')
+        if not sm.check_installed("powerpc-utils") and not sm.install("powerpc-utils"):
+            self.error('powerpc-utils is needed for the test to be run')
+        if not sm.check_installed("lsvpd") and not sm.install("lsvpd"):
+            self.error('lsvpd is needed for the test to be run')
+
+    def test(self):
+        scriptToRun = self.params.get("script", default=None)
+        toolToRun = self.params.get("tool", default=None)
+        dir = os.path.join(self.datadir, 'log')
+        if not os.path.exists(dir):
+            os.makedirs(dir)
+        commandToRun = './ras.py.data/' + scriptToRun + ' ' + \
```

Comment:
I noticed that the scripts testing RAS might be interactive, which could cause infinity hangs. Would you please add `< /dev/null` to this command to fail in case of input?